packages/nimble-components/src/rich-text-mention/users/view/styles.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from '../../../theme-provider/design-tokens';
 
 export const styles = css`
-    ${display('inline-block')}
```

Comment:
We also found another bug related to the view element where the cursor disappears when moved to the start of the line with mention. See below the GIF for the issue,
![cursor-at-start-issue](https://github.com/ni/nimble/assets/123377523/fed57eff-3679-4bd3-9f97-1c903d92bb77)

When we updated the display from `inline-block` to `inline` the issue was resolved as we were not styling any block-level properties to the element.
![cursor-at-start-fix](https://github.com/ni/nimble/assets/123377523/7c961ed5-b4a6-416d-882f-0b4f42c0903a)

We did high-level testing with this change in the storybook and it is working as expected but we will test in SLE tomorrow and once confirmed we will keep the change. If not, we will remove this change from this PR and create a separate PR to fix this bug later.

Reply:
Today we validated this change in SLE and found an issue when editing a comment where the cursor disappears after the editor transitioned from viewer. So I removed the change from this PR and created a bug to track and resolve in other follow-up PR. Bug link: https://github.com/ni/nimble/issues/1710